node-server/server.js

```diff
@@ -70,6 +72,10 @@ if (process.env.BASIC_AUTH_USERNAME && process.env.BASIC_AUTH_PASSWORD) {
   app.use(basicAuth(BASIC_AUTH_OPTIONS));
 }
 
+// Remove a few slashes
+app.use(stripTrailingSlashes);
+app.use(stripDoubleSlashes);
```

Comment:
I think this should be named something like `dedupeSlashes` because it doesn’t actually strip them

Reply:
I went with `deduplicateSlashes` 😉